### PR TITLE
Format with Black and isort

### LINKED ISSUE #1386: Unify code formatting and setup some kind of automation.
- Choose a preferred code formatter (black or yapf)
- Run it on the whole code base.
- Think of and setup some system for PRs so that the code formatter is run on all commits/all code. Possible Solutions:
	- Linter only and committer has to run himself
   - formatter bot
   - github action
   - pre-commit script

### PR DESCRIPTION
**Pull request by m3nu**
```
black .
isort .
```

Fixes #1386

- [ ] Update contributor docs

### CODE REVIEW

File: requirements.d/dev.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+black
```
Comment (real-yfprojects) [inline requirements.d/dev.txt:1]: Shouldn't isort be listed here too?

File: setup.cfg [hunk 2]
```diff
@@ -76,6 +76,7 @@ relative_files = true
 [flake8]
 ignore =
 max-line-length = 120
+extend-ignore = E203,E121,E123,E126,E226,E24,E704,W503,W504
```
Comment (real-yfprojects) [inline setup.cfg:79]: 👍

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -92,6 +92,9 @@ jobs:
         pip install .
         pip install -r requirements.d/dev.txt
     - name: Run Flake8
```
Comment (real-yfprojects) [inline .github/workflows/test.yml:94]: Maybe update name to something like `Run flake8, isort and black`

File: pyproject.toml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 [tool.black]
+line-length = 120
```
Comment (real-yfprojects) [inline pyproject.toml:2]: This should go into the previous commit `Add config files`.

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+skip-string-normalization = true
+target-version = ['py39']
+include = "src/vorta/.*.py$"
+
+[tool.isort]
+profile = "black"
+multi_line_output = 3
+lines_between_sections = 0
+src_paths = ["src", "tests"]
```
Comment (real-yfprojects) [inline pyproject.toml:11]: As noted in their [docs](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html) the parameters `skip_gitignore` and `line_length` should also be set.

**Review by real-yfprojects — CHANGES_REQUESTED** [review]
I didn't check the formatting changes :)
Some small notes below. Since you don't sign your commits these are all things I can resolve too.

**Comment by m3nu** [comment]
All fixed. Makes sense to have 2 commits here.

**Review by real-yfprojects — APPROVED** [review]

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+[tool.black]
+line-length = 120
+skip-string-normalization = true
+target-version = ['py39']
+include = "src/vorta/.*.py$"
```
Comment (real-yfprojects) [inline pyproject.toml:5]: Any reason tests aren't formatted (while isort runs on them too)?

Comment (m3nu) [reply pyproject.toml:5]: No, just an oversight on my part. We can sneak that change in with another PR.

Comment (real-yfprojects) [reply pyproject.toml:5]: I can put it into #1394.

**Pull request merged**